DOCS-9989 update to upgrade (#196)
mulesoft/docs-api-community-manager: modules/ROOT/pages/update-acm.adoc

= Upgrade API Community Manager
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:imagesdir: ../assets/images/

Keep your Anypoint API Community Manager instance up to date to take advantage of new and improved functionality in your community.

For information about upgrades, see xref:notifications.adoc[Important Notifications].

> After you install an upgrade with this process, it is not possible to undo the installation and return to a previous version. Follow this best-practice workflow:

Reviewer: ```suggestion
After you upgrade with this process, you cannot undo the installation and return to a previous version. Follow this best-practice workflow:
```

Author: That is better! Fixed.

> After you install an upgrade with this process, it is not possible to undo the installation and return to a previous version. Follow this best-practice workflow:

Reviewer: ```suggestion
After you upgrade using this process, you cannot undo the installation and return to a previous version. Follow this best-practice workflow:
```

Author: Fixed.

. Keep a partial copy sandbox or a full copy sandbox as a long-term staging environment.
+
The staging environment will have a partial or full copy of the data in the production environment.
. Create a short-term developer sandbox.
. Install the upgrade in the developer sandbox and test it.
. Discard the developer sandbox.
. Install the upgrade in the staging sandbox and test it.

> . Install the upgrade in your production environment.

Reviewer: ```suggestion
. Upgrade in your production environment.
```
Install the upgrade sounds incorrect.

Author: True! Changed to "install the new package."
